corehq/apps/app_manager/models.py

```diff
@@ -2108,13 +2108,24 @@ class CaseSearch(DocumentSchema):
     properties = SchemaListProperty(CaseSearchProperty)
     auto_launch = BooleanProperty(default=False)        # if true, skip the casedb case list
     default_search = BooleanProperty(default=False)     # if true, skip the search fields screen
-    relevant = StringProperty(default=CLAIM_DEFAULT_RELEVANT_CONDITION)
+    default_relevant = BooleanProperty(default=True)
+    additional_relevant = StringProperty()
     search_filter = StringProperty()
     search_button_display_condition = StringProperty()
     include_closed = BooleanProperty(default=False)
     default_properties = SchemaListProperty(DefaultCaseSearchProperty)
     blacklisted_owner_ids_expression = StringProperty()
 
+    def get_relevant(self):
+        relevant = self.additional_relevant or ""
+        if self.default_relevant:
+            default_condition = CaseClaimXpath(self.session_var).default_relevant()
+            if relevant:
+                relevant = f"({relevant}) and ({default_condition})"
```

Comment:
is the swapping of order here intentional? Previously the default was evaluated first (not sure if it makes any difference in the evaluation engine).

Reply:
not intended, updated in 99b8c12c792955e89feef449bb7ed647f729907d